src/plugins/components/form/buttons.js

```diff
@@ -10,7 +10,7 @@ module.exports = function () {
         cursor: 'pointer',
         display: 'inline-block',
         border: 'none',
-        fontWeight: config('theme.fontWeight.regular'),
+        fontWeight: config('theme.fontWeight.normal'),
```

Comment:
I'm replacing "regular" with "normal" everywhere when referencing the fontWeight in the theme this way. "normal" is the TW default, we added an alias "regular" so people who came from Decanter v6 (that calls font-weight: 400 "regular") can feel more at home. I'd rather go with the TW default whenever possible since we might get rid of the alias in the future.